crates/bootnode/src/main.rs

```diff
@@ -72,11 +73,13 @@ async fn main() -> anyhow::Result<()> {
     };
     let mut kad_config = kad::Config::new(dht_protocol(cli.transport.rpc.network));
     kad_config.set_replication_factor(20.try_into().unwrap());
+    let agent_info: AgentInfo = get_agent_info!();
     let behaviour = |keypair: &Keypair| Behaviour {
         identify: identify::Behaviour::new(
             identify::Config::new(ID_PROTOCOL.to_string(), keypair.public())
                 .with_interval(Duration::from_secs(60))
-                .with_push_listen_addr_updates(true),
+                .with_push_listen_addr_updates(true)
+                .with_agent_version(agent_info.to_string()),
```

Comment:
What does it do?

Reply:
When you connect to a node, you will get info which binary & version it is. By default, it's just `rust-libp2p/0.45.0` or something like that.

Reply:
This info will be printed in the peer checker result.